addons/website/models/website.py

```diff
@@ -319,8 +320,9 @@ def configurator_init(self):
             'module_state': feature.module_id.state,
         } for feature in configurator_features]
         r['logo'] = False
-        if company_id.logo and company_id.logo != company_id._get_logo():
-            r['logo'] = company_id.logo.decode('utf-8')
+        if company.logo and company.logo != company._get_logo():
+            r['logo'] = company.logo.decode('utf-8')
+            website.logo = r['logo']
```

Comment:
What is that change about?

Reply:
When company has a logo we use it as default logo for the website (and it is therefore visibile in the configurator). No attachment is created in this case. Therefore if the user doesn't change this logo in the configurator, on `configurator_apply` there will be no `logoAttachmentId`. So we need to set website logo with the one from the company and to overwrite it afterwards if the user uploaded an other logo while using the configurato. In which case there will be an attachment created and a `logoAttachmentId`.

Reply:
However it could be done in configurator_apply with something like

```python3
if not logo_attachment_id and company.logo and company.logo != company._get_logo():
    website.logo = company.logo.decode('utf-8')
```

Reply:
However it could be done in configurator_apply with something like

```python3
if not logo_attachment_id and company.logo and company.logo != company._get_logo():
    website.logo = company.logo.decode('utf-8')
```

Reply:
I think it would be clearer indeed :+1: